core/src/main/java/com/datastax/dse/driver/api/core/auth/DseGssApiAuthProviderBase.java

```diff
@@ -182,6 +183,17 @@ public Builder withLoginConfiguration(@Nullable Configuration loginConfiguration
         this.loginConfiguration = loginConfiguration;
         return this;
       }
+      /**
+       * Sets a login configuration that will be used to create a {@link LoginContext}.
+       *
+       * <p>You MUST call either this method or {@link #withSubject(Subject)}; if both are called,
+       * the subject takes precedence, and the login configuration will be ignored.
+       */
+      @NonNull
+      public Builder withLoginConfiguration(@Nullable Map<String, String> loginConfiguration) {
```

Comment:
Makes sense.

Maybe the javadoc could explain how this methods differs from the other one, something like: "Alternative to {@link #withLoginConfiguration(Configuration)}, that builds the configuration from {@code Krb5LoginModule} with the given options".